src/Tizen.NUI.Wearable/src/public/Title.cs

```diff
@@ -0,0 +1,342 @@
+        /// <summary>
+        /// The constructor of the Title class with specific Style.
+        /// </summary>
+        /// <param name="textLabelStyle">Construct Style</param>
+        /// <since_tizen> 8 </since_tizen>
+        /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Title(TextLabelStyle textLabelStyle) : base(textLabelStyle)
+        {
+            Initialize();
+        }
+
+        /// <summary>
+        /// Constructor of the Title class with special style.
+        /// </summary>
+        /// <param name="style"> style name </param>
+        /// <since_tizen> 8 </since_tizen>
+        /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Title(string style) : base(style)
+        {
+            Initialize();
+        }
+
+        /// <summary>
+        /// Sets the start and end color of the fadeout.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void FadeOutColor(Color fadeOutColor)
+        {
+            this.fadeOutColor = fadeOutColor;
+            UpdateImage();
+        }
+
+        /// <summary>
+        /// Gets or Sets the width of the fadeout effect.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public int FadeOutWidth
+        {
+            get
+            {
+                return fadeOutWidth;
+            }
+            set
+            {
+                fadeOutWidth = value;
+                UpdateImage();
+            }
+        }
+
+        /// <summary>
+        /// Dispose Title.
+        /// </summary>
+        /// <param name="type">dispose types.</param>
+        /// <since_tizen> 8 </since_tizen>
+        protected override void Dispose(DisposeTypes type)
+        {
+            if (disposed)
+            {
+                return;
+            }
+
+            if (type == DisposeTypes.Explicit)
+            {
+                if (null != leftImage)
+                {
+                    leftImage.Unparent();
+                    leftImage.Dispose();
+                    leftImage = null;
+                }
+                if (null != rightImage)
+                {
+                    rightImage.Unparent();
+                    rightImage.Dispose();
+                    rightImage = null;
+                }
+            }
+            base.Dispose(type);
+        }
+
+        private void UpdateImage()
```

Comment:
When user change size of Title Control bigger, is blur updated?

Reply:
Yes, The width is fixed and the height follows the Title Control height.